paperqa/agents/task.py

```diff
@@ -155,7 +159,13 @@ async def step(
             self.state.session.answer
         )
         if evaluation_callback := self._evaluation_callback:
+            if not isinstance(evaluation, MultipleChoiceEvaluation):
+                raise ValueError(
+                    f"Evaluation of has to be a {MultipleChoiceEvaluation.__name__}, not"
+                    f" {type(evaluation).__name__}."
```

Comment:
"Evaluation of has to be" -- this doesn't make sense

Also, can you state something about other types only being there to allow for subclassing